.github/workflows/test.yml

```diff
@@ -0,0 +1,301 @@
+name: Build, Test, and Benchmark
+
+on:
+  push:
+    branches:
+    - master
+    - issue*
+    - release*
+    paths-ignore:
+    - '.github/*'
+    - '.github/workflows/cabal.yml'
+    - '.github/workflows/haddock.yml'
+    - '.github/workflows/lint.yml'
+    - '.github/workflows/nightly.yml'
+    - '.github/workflows/stack.yml'
+    - '.github/workflows/user_manual.yml'
+    - '.github/workflows/whitespace.yml'
+    - 'src/fix-whitespace/**'
+    - '.travis'
+    - 'benchmark/**'
+    - 'doc/**'
+    - 'examples/**'
```

Comment:
Is it ok to ignore `benchmark`, `doc`, and `examples`?  These contain files belonging to the testsuite.

Reply:
Ooops, I just forgot. I put them back now.